securedrop/static/css/journalist.css

```diff
@@ -499,6 +503,40 @@ p#no-replies {
   color: #666666;
 }
 
+.tag-label {
+  display: inline-block;
+  border-bottom: 1px solid #ddd;
+  text-align: left;
+  border: 1px solid #dedede;
+  padding: 3px;
+  margin-right: 1px;
+  font-size: 8pt;
+  background-color: #7683a9;
```

Comment:
I'd consider trying to find a new color to add to the whole palette, #7683a9 is a bit overused in SD.